bot/qwen_code.py

```
@@ -0,0 +1,376 @@
+        """Ensure the MCP server is configured for Qwen Code.
+
+        Qwen Code uses `qwen mcp add` to configure MCP servers.
+        We check if 'comfytv' is already configured, and if not, add it.
+        """
+        argv = resolve_qwen_command()
+        if not argv:
+            _log.warning("[ComfyTV/bot] Cannot configure MCP: qwen executable not found")
+            return
+        try:
+            # Check if comfytv MCP server is already configured
+            result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)